src/Essentials/src/Browser/Browser.ios.cs

```diff
@@ -25,11 +25,12 @@ public async Task<bool> OpenAsync(Uri uri, BrowserLaunchOptions options)
 			return true;
 		}
 
-		[System.Runtime.Versioning.UnsupportedOSPlatform("ios11.0")]
 		private static async Task LaunchSafariViewController(Uri uri, BrowserLaunchOptions options)
 		{
 			var nativeUrl = new NSUrl(uri.AbsoluteUri);
+#pragma warning disable CA1416 // TODO: 'SFSafariViewController(NSUrl, bool)' is unsupported on: 'ios' 11.0 and later, there is an overload SFSafariViewController(NSUrl, SFSafariViewControllerConfiguration) supported from ios 11.0+
```

Comment:
Added this case to the [follow ups](https://github.com/dotnet/maui/issues/6348) list